python.d/web_log.chart.py

```diff
@@ -808,15 +809,15 @@ def check(self):
             return False
         self.storage['unique_all_time'] = list()
         self.storage['regex'] = re.compile(r'[0-9.]+\s+(?P<duration>[0-9]+)'
-                                           r' (?P<client_address>[\da-f.:]+)'
+                                           r' (?P<client_address>[\dA-Za-z.:_-]+)'
```

Comment:
`client_address`(IP address) is `[\da-f.:]+` not `[\dA-Za-z.:_-]+`
[These charts](https://github.com/firehol/netdata/blob/master/python.d/web_log.chart.py#L81-L92) will be misleading in case of hostnames instead of ip addresses.

Reply:
Sorry for sneaking host regexp there -- IP one is not useable with my log format.
I still failed to see what is so misleading in counting unique hostnames instead of IP addresses (and whole point of separate counts for unique hostnames, ipv4 and ipv6 adresses).